src/main/java/com/github/black0nion/blackonionbot/api/impl/get/Stats.java

```diff
@@ -34,11 +33,11 @@ public Object handle(Context ctx, JSONObject body, Map<String, String> headers,
 					.put("cpu_speed", CPU_MHZ))
 			.put("guild_count", StatisticsManager.getGuildCount())
 			.put("user_count", StatisticsManager.getUserCount())
-			.put("prefix", Config.prefix)
+			.put("prefix", Config.getInstance().getPrefix())
 			.put("os", OS_NAME)
```

Comment:
After the slash command pr is merged you will have a lot of issues.

Reply:
Aparently it worked well enough